rest_framework/fields.py

```diff
@@ -1522,7 +1522,10 @@ class ListField(Field):
     }
 
     def __init__(self, *args, **kwargs):
-        self.child = kwargs.pop('child', copy.deepcopy(self.child))
+        child = kwargs.pop('child', None)
+        if child is None:
+            child = copy.deepcopy(self.child)
+        self.child = child
```

Comment:
I'm a bit reluctant here to discard a possibly explicit `child=None` in favor of `self.child`.